components/post_view/post_list.jsx

```diff
@@ -493,10 +495,10 @@ export default class PostList extends React.PureComponent {
     }
```

Comment:
I'm not 100% sure how async / await work, but if they work like I think they do, this function may be a bit racy. But I don't think it's any racier than before.